packages/src/utils.ts

```diff
@@ -1440,6 +1542,23 @@ export function pkg(core: Core) {
     },
   });
 
+  pkg.createNonEventSchema({
+    name: "Unix Timestamp",
+    variant: "Exec",
+    generateIO(io) {
+      return {
+        out: io.dataOutput({
+          id: "out",
+          name: "Timestamp",
+          type: t.int(),
+        }),
+      };
+    },
+    run({ ctx, io }) {
+      ctx.setOutput(io.out, Date.now());
```

Comment:
technically this is wrong since `Date.now()` returns the time in milliseconds but unix time is in seconds

Reply:
renamed node to "Current Timestamp (ms)"